modules/ROOT/pages/tutorials/create-auradb-instance-from-terminal.adoc

```diff
@@ -0,0 +1,95 @@
+[[create-auradb-instance-in-terminal]]
+= Create an AuraDB instance from the terminal
+:description: This tutorial describes using the terminal to create an instance in the Aura Console. 
+
+This tutorial describes using the terminal to create an instance in the Aura Console. 
+
+== Preparation
+
+=== Generate API credentials
+
+* Log in to the Aura Console
+* Click your email address in the top right corner and select *Account details*.
+* In the *API credentials* section, select *Create*. 
+Enter a descriptive name and save the generated Client ID and Client Secret.
+
+=== cURL
+* Install cURL via your terminal
+* For macOS with Homebrew: Type `brew install curl` in your terminal.
```

Comment:
```suggestion
* For macOS with Homebrew: use `brew install curl`.
```
But what about Windows? Linux? 
Also, these bullets are not steps as the previous section is.